### DISCUSSION TITLE
Custom fonts

### DISCUSSION THREAD

**Discussion by Zabooby** [excalith/excalith-start-page#40] [Q&A] [answered]
I'm using Vercel to run the startpage, to change the font, do I just change [this](https://github.com/Zabooby/excalith-start-page/blob/d3fb3c3995f18f8e764219618b901e75e96411e8/src/styles/globals.css#L46) line? Does this cover all text in the startpage?

**Comment by excalith** [comment, marked as answer]
Hey @Zabooby, you need to change more than that.

- You need to import the font through fontsource as a dependency
- You need to import the required modules in index,js
- You need to update the tailwing.config.js with the new font
- Adding the font on global.css body won't hurt as well

Please have a look at [this search](https://github.com/search?q=repo%3Aexcalith%2Fexcalith-start-page%20font&type=code) regarding some examples of what to do
For more information, please have a look at the [Fontsource](https://fontsource.org)
